packages/automerge-repo/src/synchronizer/DocSynchronizer.ts

```diff
@@ -69,13 +71,9 @@ export class DocSynchronizer
         `[${this.handle.documentId}]->[${peerId}]: sendSyncMessage: ${message.byteLength}b`,
         decoded
       )
-      conciseLog(
-        `📣 sendSyncMessage\t${JSON.stringify({
-          documentId: this.handle.documentId,
-          peerId,
-          messageLength: message.byteLength,
-        })}`
-      )
+
+      conciseLog(`${peerId} ⬅️️  ${this.handle.documentId} (${message.byteLength} bytes)`)
+      opsLog(getChangesOfMessage(decoded))
```

Comment:
I believe this will execute (and can be quite expensive) even if the logging is not enabled. you should guard it. (the debug library has instructions for how to check that)